numba/cuda/extending.py

```diff
@@ -2,6 +2,9 @@
 Added for symmetry with the core API
 """
 
+from functools import partial
 from numba.core.extending import intrinsic as _intrinsic
+from numba.core.extending import overload as _overload
 
 intrinsic = _intrinsic(target='cuda')
```

Comment:
Should I have used `functools.partial` for `intrinsic` as well?

Reply:
Is it worth doing `functools.update_wrapper` on these so as to inherit the docstrings etc?